pages/docs/cloud.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Grouparoo Cloud Integration"
+date: "2021-11-08"
+pullQuote: "An easy-to-follow guide on integrating your Code Config repository with Grouparoo Cloud"
+---
+
+We're excited to share our Cloud offering! This guide should cover everything you need to get integrated
+using your own repository with Grouparoo Cloud.
+
+### Step 1 - Getting to your Project
+
+Sign in to your [Cloud](http://cloud.grouparoo.com/) account, or create a new one.
+
+#### If a new account
+
+Follow the steps shown on Cloud to create an Organzation and Project.
+
+#### If an existing account, but a new Grouparoo Project
+
+Use the Project dropdown to create a new Project.
+screenshot here
+
+#### If an existing account, and project
+
+You're set, follow the steps below!
+
+### Step 2 - Acquring an API Key
+
+Get your API Key. This can be found on your Project's primary page by clicking the Key icon near
+the top right of the page.
+API Keys are created with your project! Press the Copy button to copy the API Key.
```

Comment:
Steps to create a project / apikey would be great to have, but we don't really support it quite yet. I think we should probably only put things in the docs that are available to avoid confusion, and assume that if they are setting up their repo then they already have a project set up.